src/lib/repository.js

```diff
@@ -23,7 +23,7 @@ function getRemoteUrl (pkg, callback) {
   }
   if (/^git\+/.test(pkg.repository.url)) {
     pkg.repository.url = pkg.repository.url.substr(4)
-    pkg.repository.url = pkg.repository.url.replace(/^ssh:\/\/git@/, 'https://')
+    pkg.repository.url = pkg.repository.url.replace(/^(ssh:\/\/git|git:\/\/)@/, 'https://')
```

Comment:
Actually, it never even reaches here because the url doesn't start with `git+`. Parsing the url and changing the protocol is probably more robust. I'll close this PR and start again.